db/models/models.js

```diff
@@ -88,3 +88,31 @@ exports.insertCommentByReviewId = (review_id, newComment) => {
       return res.rows[0];
     });
 };
+
+exports.updateVotes = (review_id, inc_votes) => {
+  if (!inc_votes) {
+    return Promise.reject({
+      status: 400,
+      msg: "Bad request"
+    });
+  }
+  return checkExists("reviews", "review_id", review_id)
+    .then(() => {
+      return db.query(
+        `UPDATE reviews
+      SET votes = votes + $1
+      WHERE review_id = $2
+      RETURNING*;`,
+        [inc_votes, review_id]
+      );
+    })
+    .then((result) => {
+      const review = result.rows[0];
+      if (!review) {
```

Comment:
This is where you could check if the review didn't exist - rather than something invalid about the request